tools/flake8/config.ini

```diff
@@ -1,6 +1,8 @@
+[flake8]
 filename = *.py,*.py.in
 show-source = True
 statistics = True
 count = True
-ignore = F403
-max-line-length = 79
+ignore = F401,F403,F405
+max-line-length = 90
+exclude = .git,__pycache__,build,cmake
```

Comment:
Why is `cmake` directory excluded? It includes `.py.in` files which need to be checked.